### DISCUSSION TITLE
Hope we could support better display for contact info card on dark mode

### DISCUSSION THREAD

**Discussion by quietcoolwu** [protz/thunderbird-conversations#1871] [Ideas]
Hi @Standard8,
Thanks for your contribution on this extension,  really helped the efficiency on handling back&forth email threads.

Just one thing, I found the UI display use contrast display for mail contact info, which is not very friendly for dark mode under thunderbird v102+, could we work on some improve? Thanks!

![image](https://user-images.githubusercontent.com/11589421/179444790-b14fc547-7249-4813-8917-a47b96bad742.png)

**Comment by Standard8** [comment]
I've just fixed this in #1883 - though it may take a week or so before it gets released.
